src/i18n.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @flow
+ * @file Exports language, messages and react-intl language data for internationalization
+ * @author Box
+ */
+
+import localeData from 'react-intl-locale-data'; // eslint-disable-line
```

Comment:
do we need to import all locale-data from react-intl if box-locales only supports a subset of them?

Reply:
Not sure how to specifically filter out only used languages. This was modified off of how all the other repos at box import locale-data

Reply:
I guess it would be interesting to see how much this impacts the overall bundle size, when I run a `du -h .` on the react-intl/locale-data directory, all the files add up to 1.3MB. My guess would be that would be a direct impact on the bundle size since it already seems minified and obfuscated. 

Did a quick StackOverflow search and saw this link: https://stackoverflow.com/a/41287219 Maybe there's something we could do here like that for only the locales supported by box-locales?